### ISSUE TITLE
Inject historical Slack threads into Scoold as questions and answers

### ISSUE THREAD

**Issue by albogdano** [Erudika/scoold-pro#55]
... using the "Ask on Scoold" button

**Comment by albogdano** [comment]
Done! Added for Slack, MS Teams and Mattermost. On Slack and MS Teams, clicking the "Ask on Scoold" button will save the full thread. On Mattermost you will have to first copy the thread link, then run the `/scoold save {thread_link}` command.

**Comment by shai-katzir-hs** [comment]
Hello, when running "ask on Scoold" from a thtread nothing happens. Also, `save` seems to not be supported by slack plugin

**Comment by albogdano** [comment]
The feature requires that Slack has a direct connection to your Scoold server. You can test it locally using a tool like ngrok. The command above has a typo - it's not `/slack save` but `/scoold save` and it only works with the Mattermost integration.

**Comment by albogdano** [comment]
@shai-katzir-hs After some investigation, I found that there was indeed an issue preventing the save thread functionality. I updated a few dependencies and fixed the code which threw exceptions. Another thing I found is that the Scoold bot is required to be a member of the Slack channel where the action takes place. If that is not the case, try inviting the bot to the channel with this command:
```
/invite @scoold
```
The fix will be available in the next iteration of Scoold Pro.
